scripts/create_issue.py

```
@@ -0,0 +1,160 @@
+    # Get the current main/master branch SHA
+    print("Getting main branch SHA...")
+    sha = get_main_branch_sha(owner, repo)
+
+    # Create branch in remote
+    print("Creating branch in remote...")
+    create_branch(owner, repo, branch_name, sha, args.token or os.getenv("GITHUB_TOKEN"))
+
+    # Fetch and checkout
+    print("Fetching from origin...")
+    try:
+        subprocess.run(["git", "fetch", "origin"], check=True)
```

Ruff (0.15.17)

[error] Starting a process with a partial executable path

(S607)